components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -180,36 +180,48 @@ def _process_s3_input(
         paths_to_compress_buffer.add_file(object_metadata)
 
 
-def _write_failed_path_log(
-    invalid_path_messages: List[str], logs_directory: Path, job_id: Any
+def _write_user_failure_log(
+    content: Union[str, List[str]],
```

Comment:
We could just update `poll_running_jobs` to generate a list rather than a string, right?